.github/workflows/build_and_test.yml

```
@@ -5,26 +5,32 @@ on:
 jobs:
   build:
-
     runs-on: ubuntu-latest
     
     strategy:
       matrix:
-        php-version: [ '7.4', '8.0', '8.1', '8.2', '8.3' ]
+        php-version: [ '7.4', '8.0', '8.1', '8.2', '8.3', '8.4' ]
 
     steps:
     - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)